examples/text_labels_and_annotations/font_table_sgskip.py

```diff
@@ -0,0 +1,106 @@
+"""
+==========
+Font table
+==========
+
+Matplotlib's font support is provided by the FreeType library.
+
+Here, we use `~.Axes.table` build a font table that shows the glyphs by Unicode
+codepoint, and print the glyphs corresponding to codepoints beyond 0xff.
+
+Usage::
+
+    python font_table_sgskip.py /path/to/font/file
+"""
+
+import unicodedata
+
+from matplotlib import (
```

Comment:
Is that really common style? I would have used
~~~
import matplotlib.pyplot as plt
import matplotlib.font_manager as fm
~~~